source/opt/set_spec_constant_default_value_pass.h

```diff
@@ -43,6 +43,39 @@ class SetSpecConstantDefaultValuePass : public Pass {
   const char* name() const override { return "set-spec-const-default-value"; }
   bool Process(ir::Module*) override;
 
+  // Parses the given string to get a mapping from Spec Id to default value
+  // strings. Returns a unique pointer of the mapping from spec ids to spec
+  // constant default value strings built from the given |str| on success. If
+  // the given string is not valid for building the mapping, returns a nullptr.
+  // A valid string for building the mapping should follow the below example:
+  //
+  //  { <spec id A> : <default value for A>, <spec id B> : <default value for
+  //  B>, ...}
+  //
+  //  The string must starts with '{', and should ends with '}'. However the
+  //  missing the ending '}' may still be valid for parsing.
```

Comment:
Let's be rigid. Missing `}` should be considered invalid. Allowing special cases may bite us in unexpected way in the future.